quipucords/scanner/openshift/test_entities.py

```diff
@@ -145,3 +148,76 @@ class TestClass(OCPBaseEntity):
 
         entity = TestClass(name="foo")
         assert entity.kind is not None
+
+
+@pytest.mark.parametrize(
+    "value, expected_result",
+    [
+        ("200m", 0.2),
+        ("3500m", 3.5),
+        (2.0, 2.0),
+        (4, 4.0),
```

Comment:
please add another parameter here:

something like `("2", 2.0),`

Reply:
@bruno-fs  Adding this specific pair (value =" 2," expected_value =2.0) will fail the first assertion because unless value it's a string and contains a millicore suffix, it won't be treated here, it will be returned to pydantic to perform its magic. 

This fails because the function  _convert_cpu_value won't do anything with "2"
```
    converted_value = NodeResources._convert_cpu_value("2")
    assert converted_value == 2.0
```
This will be successful: 
```
    node_resource = NodeResources(cpu="2")
    assert node_resource.cpu == 2.0
```
If u want me to add this use case I can split the test_cpu_validator_with_convertible_values in two different tests, in one I test if values can be converted, in the other I test final values of attributes.

Reply:
Good point.
I think we have 2 options here:
- ensure "2" will bypass the validation `("2", "2")`
- change the test to instantiate NodeResources with cpu attribute (it'd cover what we need).

I prefer the first option, but do whatever you prefer.